dll/win32/browseui/shellfind/CSearchBar.cpp

```diff
@@ -131,6 +136,34 @@ LRESULT CSearchBar::OnSearchButtonClicked(WORD wNotifyCode, WORD wID, HWND hWndC
         return 0;
     }
 
+    // See if we have a szFileName, our first character is not an asterisk,
+    // and we have room in MAX_PATH to add an asterisk at the beginning of szFileName then add it
+    if (wcscmp(pSearchStart->szFileName, L"") != 0)
+    {
+        len = wcslen(pSearchStart->szFileName);
+        if(len > 0)
+        {
+            ptrchar = pSearchStart->szFileName[len - 1];
+            endchar = ptrchar;
```

Comment:
`ptrchar` seems pointless, since you can just assign `startchar` and `endchar` directly.

Reply:
Removed `ptrchar` as you suggested. Thanks.